src/main/resources/com/arcbees/website/server/html/home.vm

```diff
@@ -42,22 +42,14 @@
         }
     </style>
 
+    <script>
```

Comment:
So ... only a stripped down version of the analytics experiment code, that you trigger manually?